src/core/jsonpointer/include/sourcemeta/core/jsonpointer.h

```diff
@@ -1,17 +1,16 @@
 #ifndef SOURCEMETA_CORE_JSONPOINTER_H_
 #define SOURCEMETA_CORE_JSONPOINTER_H_
 
+#include "sourcemeta/core/json_hash.h"
```

Comment:
Can you avoid `"..."` includes in favour of `<...>` includes?

Reply:
Also, there is an ordering of includes in most of these files. You can notice that the Sourcemeta includes are below

Reply:
> Can you avoid `"..."` includes in favour of `<...>` includes?

Why is `<... >` preferred for non C++ standard headers?

Reply:
`"..."` depends on position, while `<...>` depends on include paths. With the latter, it is easier to control for more advanced setups